toggl_webhooks.py

```diff
@@ -65,10 +65,15 @@ async def message_handler_outer(action, model, msg: TogglSocketMessage):
 # The actual handler; in this case, it just sends the data to the server in the manner defined by
 # the config file.
 async def message_handler(action: str, model: str, method: str, url: str, msg: TogglSocketMessage,
-                          trials=None, **request_kwargs):
+                          trials=None, auth=None, **request_kwargs):
     trials = trials or 1
     assert trials >= 1
     trial = 0
+    if auth:
+        if isinstance(auth, list):
+            # Basic Authorization support (user:password)
+            auth = tuple(auth)
+        request_kwargs['auth'] = auth
```

Comment:
@ihoru Nice idea! Just to confirm - you have tested this, right? Since the WS is an internal API we don't have any documentation to guarantee that this will work without testing it

Reply:
1. This is about requesting webhooks, not WS (so it's normal `request.get(auth=(name, pass))` behavior)
2. Yes, I tested it.

Reply:
Ah you're right, sorry.. wasn't paying enough attention.